akka-javasdk-testkit/src/main/java/akka/javasdk/testkit/EventSourcedTestKit.java

```diff
@@ -26,14 +27,26 @@
 public class EventSourcedTestKit<S, E, ES extends EventSourcedEntity<S, E>>
     extends EventSourcedEntityEffectsRunner<S, E> {
 
-  private final ES entity;
   private final String entityId;
 
   private final JsonMessageCodec messageCodec;
 
+  public static final String DEFAULT_TEST_ENTITY_ID = "testkit-entity-id";
+
   private EventSourcedTestKit(ES entity, String entityId) {
     super(entity);
-    this.entity = entity;
+    this.entityId = entityId;
+    this.messageCodec = new JsonMessageCodec();
```

Comment:
Is that messageCodec even used? It has been removed from the EventSourcedTestKit in the java-spi branch.